puppet/zulip_ops/files/zulip-ec2-configure-interfaces

```diff
@@ -55,13 +55,17 @@ import boto.utils
 import netifaces
 from six.moves import range
 
+from typing import Union
```

Comment:
So, this isn't a problem with the other scripts, but this particular script is designed to run with very little in the way of dependencies installed, and the `typing` module isn't installed by default with Python 2.  So, you should put this line inside and `if False:`, so that the typing import will only be used by the type checker, and won't be attempted at runtime (which would break the script).